_posts/2016-08-17-user-stories.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Giulia Mantuano
+comments: true
+date: 2016-08-17 14:25:09+00:00
+layout: post
+asset-type: post
+slug: hello
+title: Improve team's communication with User Stories
+image:
+    src: /assets/img/custom/blog/user-stories-sketchnote.jpg
+category: agile
+tags:
+- scrum
+- user stories
+- team
+---
+
+The challenge of Software development is about building a product which fulfills both business and users expectations. 
+
+The focus for each role within an Agile team is different: Project Managers want to see progress and want a quality product that is in time and value for money; Product Manager wants flexibility; Testers want to measure; Developers wants to write outstanding and sustainable code; Users are looking for a product which help accomplishing their tasks. How is it possible to find an agreement between all of these needs and build a product which could be a good compromise between building the right thing, building the thing right and building it in time?
+
+In order to build a shared understanding about the product vision, goals and values it is necessary to have every team member involved in the process (real users included!). User Stories are a tool that help accomplish that: they improve communication within teams and stakeholders and reduce the risk of misinterpreting product development requirements.
```

Comment:
In order to build a shared understanding **of** the product vision, goals and values it is necessary to have every team member involved in the process (real users included!). User Stories are a tool that **helps** accomplish that: they improve communication within teams and stakeholders and reduce the risk of misinterpreting product development requirements.